airflow/models/taskinstance.py

```diff
@@ -1111,6 +1111,7 @@ def check_and_change_state_before_execution(
         if not test_mode:
             session.add(Log(State.RUNNING, self))
         self.state = State.RUNNING
+        self.pid = None
```

Comment:
Should this be up with line 1043-1044 instead?

Reply:
Yes, i put it back on line where it was in 2.0 , but I'm going to push a commit to put it next to the hostname which I think is a good place too. https://github.com/apache/airflow/blob/2.0.2/airflow/models/taskinstance.py#L1076